flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/ResultPartitionTest.java

```diff
@@ -399,6 +399,10 @@ void testIdleAndBackPressuredTime() throws IOException, InterruptedException {
         while (!isInBlockingBufferRequest(requestThread.getStackTrace())) {
             Thread.sleep(50);
         }
+        // there is an extreme case where the request thread recovers from blocking very
+        // quickly, resulting in a calculated back-pressure time is equal to 0. This is used to
+        // avoid this case.
+        Thread.sleep(5);
```

Comment:
We can also change `isGreaterThan(0)` to `isGreaterThanOrEqualTo(0)`, but this seems to break our original intention of test the back pressure time is not zero.